build/Versions.props

```diff
@@ -1,10 +1,10 @@
 <Project>
   <PropertyGroup>
-    <VersionBase>2.3.0</VersionBase>
-    <PreReleaseVersionLabel>beta2</PreReleaseVersionLabel>
+    <VersionBase>2.6.0</VersionBase>
```

Comment:
@jasonmalinowski to comment, but Roslyn should be moving to 2.5.0 soon, to match the VS version. Analyzers might want to do the same.